internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -158,7 +167,146 @@ func (db *gormdb) Migrate(ctx context.Context) error {
 	}
 
 	migrationsDir := db.cfg.Type
-	return goose.Up(client, migrationsDir)
+	if err := goose.UpContext(ctx, client, migrationsDir); err != nil {
+		return fmt.Errorf("goose up: %w", err)
+	}
+
+	if err := db.backfillUsersAndOrgs(ctx); err != nil {
+		return fmt.Errorf("backfill users and orgs: %w", err)
+	}
+
+	return nil
+}
+
+func (db *gormdb) backfillUsersAndOrgs(ctx context.Context) error {
```

Comment:
1. Document this function *in detail*, and add a TODO with a ticket ID for removing it
2. If this function runs only when AK does a migration, it actually won't run in production unless we run it explicitly, so what are you gaining here?

Reply:
1. done
2. we need to discuss proper procedures re prod here, out of scope for oss.